src/Components/Navibar.js

```diff
@@ -1,8 +1,9 @@
 import { hover } from "@testing-library/user-event/dist/hover";
-import React from "react";
-import { Navbar, Nav, Button, Container } from 'react-bootstrap';
+import React, { useState } from "react";
+import { Navbar, Nav, Button, Container, Form } from 'react-bootstrap';
```

Comment:
@Oleg-pnu please stick to one style of quotes (single or double) throughout the project.